### ISSUE TITLE
Tribe payments

### ISSUE THREAD

**Issue by tomastiminskas** [stakwork/sphinx-rs#60]
On V1 we supported tribe payments. Basically a member of a tribe could send a payment to another member of the tribe. The tribe admin uses the `message_uuid` sent in the `sendPayment` endpoint to identify the message and the sender of that original message and forward the sats. 

The message is created with `recipient_alias` and `recipient_pic` so the UI can show who sent it and to whom it was sent. 

Not sure if we can support this on V2, but I found that all the apps were allowing to tap on a message profile pic and the app was presenting the SEND SATS popup (iOS and Mac have the logic disabled, but I think Android has it enabled since I saw a tribe payment in Planet Sphinx and the UI is broken since recipient Alias and Pic are null)

For now I will disabled tapping on profile picture, but would be good to add the needed logic to add this feature if possible.

@Evanfeenstra let me know your thoughts

**Comment by Evanfeenstra** [comment]
This is not yet supported on v2 tribes server. Is it just a `DirectPayment` (type 5)?

Did you mean that the admin uses the `reply_uuid` to identify the recipient? @tomastiminskas

**Comment by tomastiminskas** [comment]
Yes, it's a direct payment (type = 5). The admin uses the `reply_uuid` to identify the recipient and forward the sats and send a forwards the type 5 message to the rest of the members adding `recipient_alias` and `recipient_pic` so the members can show the recipient in the UI
